Add project haystack example to propsal (#430)
BuildingSync/schema: proposals/2022/BuildingSync Project Haystack Example.md

# BuildingSync Project Haystack Example

> The files in this directory contain an example implementation of connecting a BuildingSync model (.xml) to a [Project Haystack](https://project-haystack.org/) model (.json). The files are _not_ valid schema representations and are shortened to highlight the relevant common elements and entities.

Reviewer: ```suggestion
The examples below demonstrate an implementation for connecting a BuildingSync model (.xml) to a [Project Haystack](https://project-haystack.org/) model (.json). The files are _not_ valid schema representations and are shortened to highlight the relevant common elements and entities.
```

Author: Good catch, I've address these changes throughout.
